 BinaryConverter/src/harryPotterReader/UniqueWord.java

```diff
@@ -0,0 +1,75 @@
+package harryPotterReader;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class UniqueWord {
+	
+	private ArrayList<String> uniqueList = new ArrayList<String>();
+	
+	private ArrayList<Integer> count = new ArrayList<Integer>();
+	
+	
+	public UniqueWord(File f) {
+		
+		try {
+			
+			Scanner s = new Scanner(f);
+			System.out.println("file open");
+			
+			
+			//System.out.println(s.next().toLowerCase());
+			
+			while(s.hasNext()) {
+				
+				String newWord = s.next().toLowerCase();
+				
+				if(newWord.charAt(0) == '\"' 
+						|| newWord.charAt(0) == '(') {
+					newWord = newWord.substring(1); // remove first letter if punctuation
+				} else if (newWord.charAt(newWord.length()-1) == '.' 
+						|| newWord.charAt(newWord.length()-1) == ','
+						|| newWord.charAt(newWord.length()-1) == '?'
+						|| newWord.charAt(newWord.length()-1) == '!'
+						|| newWord.charAt(newWord.length()-1) == '\"'
+						|| newWord.charAt(newWord.length()-1) == ')') {
```

Comment:
GOOD. It probably doesn't include all the cases. Use the method `Character.isLetter(char a)`